<?php

    include '../../../common/include/common.php';
    include_once '../../config.php';
    $module = modules::GATK;
    printHeader($module, $name, topSN::guide);
?>

<link type='text/css' rel='stylesheet' href='gatkDoc.css'>

<div class='row-fluid' id="top">

	
	<?php $group = 'Metrics'; ?>

	<section class="span4">
		<aside class="well">
			<a href="index.html"><h4><i class='fa fa-chevron-left'></i> Back to Tool Docs Index</h4></a>
		</aside>
		<?php getForumPosts( 'HsMetrics' ) ?>

	</section>

	<div class="span8">

		<h1>HsMetrics</h1>

		<p class="lead">Metrics</p>

			<h3>Category
				<small> Metrics</small>
			</h3>
		<hr>
		<h2>Overview</h2>
		<p>Metrics generated by CollectHsMetrics for the analysis of target-capture sequencing experiments. The metrics
 in this class fall broadly into three categories:</p>

 <ul>
     <li>Basic sequencing metrics that are either generated as a baseline against which to evaluate other
     metrics or because they are used in the calculation of other metrics.  This includes things like
     the genome size, the number of reads, the number of aligned reads etc.</li>
     <li>Metrics that are intended for evaluating the performance of the wet-lab assay that generated the data.
     This group includes metrics like the number of bases mapping on/off/near baits, %selected, fold 80 base
     penalty, hs library size and the hs penalty metrics. These metrics are calculated prior to some of the
     filters are applied (e.g. low mapping quality reads, low base quality bases and bases overlapping in the middle
     of paired-end reads are all counted).
     </li>
     <li>Metrics for assessing target coverage as a proxy for how well the data is likely to perform in downstream
     applications like variant calling. This group includes metrics like mean target coverage, the percentage of bases
     reaching various coverage levels, and the percentage of bases excluded by various filters. These metrics are computed
     using the strictest subset of the data, after all filters have been applied.</li>
 </ul>
@author Tim Fennell

				<p>This table summarizes the values that are specific to this metric.</p>
				<table class="table table-striped table-bordered table-condensed">
					<thead>
					<tr>
						<th>Metric</th>
						<th>Summary</th>
					</tr>
					</thead>
					<tr>
						<td>BAIT_SET<br />
						<td>The name of the bait set used in the hybrid selection.</td>
					</tr>
					<tr>
						<td>BAIT_TERRITORY<br />
						<td>The number of bases which are localized to one or more baits.</td>
					</tr>
					<tr>
						<td>BAIT_DESIGN_EFFICIENCY<br />
						<td>The ratio of TARGET_TERRITORY/BAIT_TERRITORY.  A value of 1 indicates a perfect design efficiency, while a valud of 0.5 indicates that half of bases within the bait region are not within the target region.</td>
					</tr>
					<tr>
						<td>ON_BAIT_BASES<br />
						<td>The number of PF_BASES_ALIGNED that are mapped to the baited regions of the genome.</td>
					</tr>
					<tr>
						<td>NEAR_BAIT_BASES<br />
						<td>The number of PF_BASES_ALIGNED that are mapped to within a fixed interval containing a baited region, but not within the baited section per se.</td>
					</tr>
					<tr>
						<td>OFF_BAIT_BASES<br />
						<td>The number of PF_BASES_ALIGNED that are mapped away from any baited region.</td>
					</tr>
					<tr>
						<td>PCT_SELECTED_BASES<br />
						<td>The fraction of PF_BASES_ALIGNED located on or near a baited region (ON_BAIT_BASES + NEAR_BAIT_BASES)/PF_BASES_ALIGNED.</td>
					</tr>
					<tr>
						<td>PCT_OFF_BAIT<br />
						<td>The fraction of PF_BASES_ALIGNED that are mapped away from any baited region, OFF_BAIT_BASES/PF_BASES_ALIGNED.</td>
					</tr>
					<tr>
						<td>ON_BAIT_VS_SELECTED<br />
						<td>The fraction of bases on or near baits that are covered by baits, ON_BAIT_BASES/(ON_BAIT_BASES + NEAR_BAIT_BASES).</td>
					</tr>
					<tr>
						<td>MEAN_BAIT_COVERAGE<br />
						<td>The mean coverage of all baits in the experiment.</td>
					</tr>
					<tr>
						<td>PCT_USABLE_BASES_ON_BAIT<br />
						<td>The number of aligned, de-duped, on-bait bases out of the PF bases available.</td>
					</tr>
					<tr>
						<td>PCT_USABLE_BASES_ON_TARGET<br />
						<td>The number of aligned, de-duped, on-target bases out of all of the PF bases available.</td>
					</tr>
					<tr>
						<td>FOLD_ENRICHMENT<br />
						<td>The fold by which the baited region has been amplified above genomic background.</td>
					</tr>
					<tr>
						<td>HS_LIBRARY_SIZE<br />
						<td>The estimated number of unique molecules in the selected part of the library.</td>
					</tr>
					<tr>
						<td>HS_PENALTY_10X<br />
						<td>The "hybrid selection penalty" incurred to get 80% of target bases to 10X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 10X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 10 * HS_PENALTY_10X.</td>
					</tr>
					<tr>
						<td>HS_PENALTY_20X<br />
						<td>The "hybrid selection penalty" incurred to get 80% of target bases to 20X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 20X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 20 * HS_PENALTY_20X.</td>
					</tr>
					<tr>
						<td>HS_PENALTY_30X<br />
						<td>The "hybrid selection penalty" incurred to get 80% of target bases to 30X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 30X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 30 * HS_PENALTY_30X.</td>
					</tr>
					<tr>
						<td>HS_PENALTY_40X<br />
						<td>The "hybrid selection penalty" incurred to get 80% of target bases to 40X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 40X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 40 * HS_PENALTY_40X.</td>
					</tr>
					<tr>
						<td>HS_PENALTY_50X<br />
						<td>The "hybrid selection penalty" incurred to get 80% of target bases to 50X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 50X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 50 * HS_PENALTY_50X.</td>
					</tr>
					<tr>
						<td>HS_PENALTY_100X<br />
						<td>The "hybrid selection penalty" incurred to get 80% of target bases to 100X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 100X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 100 * HS_PENALTY_100X.</td>
					</tr>
					</tbody>
				</table>

        <hr>
        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
        <hr>
        <p class="see-also">See also 
        	<a href="https://software.broadinstitute.org/gatk/documentation/">General Documentation</a> |
        	<a class="hide_me_php" href="index.html">Tool Docs Index</a> <a class="hide_me_html" href="index">Tool Documentation Index</a> |
        	<a href="https://gatkforums.broadinstitute.org/gatk/">Support Forum</a>
        </p>

        <p class="version">GATK version 4.5.0.0 built at Wed, 13 Dec 2023 17:42:57 -0500.
        </p>

	</div>

	<?php printFooter($module); ?>