<?php

    include '../../../common/include/common.php';
    include_once '../../config.php';
    $module = modules::GATK;
    printHeader($module, $name, topSN::guide);
?>

<link type='text/css' rel='stylesheet' href='gatkDoc.css'>

<div class='row-fluid' id="top">

	
	<?php $group = 'Metrics'; ?>

	<section class="span4">
		<aside class="well">
			<a href="index.html"><h4><i class='fa fa-chevron-left'></i> Back to Tool Docs Index</h4></a>
		</aside>
		<?php getForumPosts( 'FingerprintingDetailMetrics' ) ?>

	</section>

	<div class="span8">

		<h1>FingerprintingDetailMetrics</h1>

		<p class="lead">Metrics</p>

			<h3>Category
				<small> Metrics</small>
			</h3>
		<hr>
		<h2>Overview</h2>
		Detailed metrics about an individual SNP/Haplotype comparison within a fingerprint comparison.
@author Tim Fennell

				<p>This table summarizes the values that are specific to this metric.</p>
				<table class="table table-striped table-bordered table-condensed">
					<thead>
					<tr>
						<th>Metric</th>
						<th>Summary</th>
					</tr>
					</thead>
					<tr>
						<td>READ_GROUP<br />
						<td>The sequencing read group from which sequence data was fingerprinted.</td>
					</tr>
					<tr>
						<td>SAMPLE<br />
						<td>The name of the sample who's genotypes the sequence data was compared to.</td>
					</tr>
					<tr>
						<td>SNP<br />
						<td>The name of a representative SNP within the haplotype that was compared. Will usually be the
 exact SNP that was genotyped externally.</td>
					</tr>
					<tr>
						<td>SNP_ALLELES<br />
						<td>The possible alleles for the SNP.</td>
					</tr>
					<tr>
						<td>CHROM<br />
						<td>The chromosome on which the SNP resides.</td>
					</tr>
					<tr>
						<td>POSITION<br />
						<td>The position of the SNP on the chromosome.</td>
					</tr>
					<tr>
						<td>EXPECTED_GENOTYPE<br />
						<td>The expected genotype of the sample at the SNP locus.</td>
					</tr>
					<tr>
						<td>OBSERVED_GENOTYPE<br />
						<td>The most likely genotype given the observed evidence at the SNP locus in the sequencing data.</td>
					</tr>
					<tr>
						<td>LOD<br />
						<td>The LOD score for OBSERVED_GENOTYPE vs. the next most likely genotype in the sequencing data.</td>
					</tr>
					<tr>
						<td>OBS_A<br />
						<td>The number of observations of the first, or A,  allele of the SNP in the sequencing data.</td>
					</tr>
					<tr>
						<td>OBS_B<br />
						<td>The number of observations of the second, or B, allele of the SNP in the sequencing data.</td>
					</tr>
					</tbody>
				</table>

        <hr>
        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
        <hr>
        <p class="see-also">See also 
        	<a href="https://software.broadinstitute.org/gatk/documentation/">General Documentation</a> |
        	<a class="hide_me_php" href="index.html">Tool Docs Index</a> <a class="hide_me_html" href="index">Tool Documentation Index</a> |
        	<a href="https://gatkforums.broadinstitute.org/gatk/">Support Forum</a>
        </p>

        <p class="version">GATK version 4.5.0.0 built at Wed, 13 Dec 2023 17:42:57 -0500.
        </p>

	</div>

	<?php printFooter($module); ?>