<?php

    include '../../../common/include/common.php';
    include_once '../../config.php';
    $module = modules::GATK;
    printHeader($module, $name, topSN::guide);
?>

<link type='text/css' rel='stylesheet' href='gatkDoc.css'>

<div class='row-fluid' id="top">

	
	<?php $group = 'Metrics'; ?>

	<section class="span4">
		<aside class="well">
			<a href="index.html"><h4><i class='fa fa-chevron-left'></i> Back to Tool Docs Index</h4></a>
		</aside>
		<?php getForumPosts( 'GenotypeConcordanceDetailMetrics' ) ?>

	</section>

	<div class="span8">

		<h1>GenotypeConcordanceDetailMetrics</h1>

		<p class="lead">Metrics</p>

			<h3>Category
				<small> Metrics</small>
			</h3>
		<hr>
		<h2>Overview</h2>
		Class that holds detail metrics about Genotype Concordance
@author George Grant

				<p>This table summarizes the values that are specific to this metric.</p>
				<table class="table table-striped table-bordered table-condensed">
					<thead>
					<tr>
						<th>Metric</th>
						<th>Summary</th>
					</tr>
					</thead>
					<tr>
						<td>VARIANT_TYPE<br />
						<td>The type of the event (i.e. either SNP or INDEL)</td>
					</tr>
					<tr>
						<td>TRUTH_SAMPLE<br />
						<td>The name of the 'truth' sample</td>
					</tr>
					<tr>
						<td>CALL_SAMPLE<br />
						<td>The name of the 'call' sample</td>
					</tr>
					<tr>
						<td>TRUTH_STATE<br />
						<td>The state of the 'truth' sample (i.e. HOM_REF, HET_REF_VAR1, HET_VAR1_VAR2...)</td>
					</tr>
					<tr>
						<td>CALL_STATE<br />
						<td>The state of the 'call' sample (i.e. HOM_REF, HET_REF_VAR1...)</td>
					</tr>
					<tr>
						<td>COUNT<br />
						<td>The number of events of type TRUTH_STATE and CALL_STATE for the EVENT_TYPE and SAMPLEs</td>
					</tr>
					<tr>
						<td>CONTINGENCY_VALUES<br />
						<td>The list of contingency table values (TP, TN, FP, FN) that are deduced from the truth/call state comparison, given the reference.

 In general, we are comparing two sets of alleles.  Therefore, we can have zero or more contingency table values represented in one comparison.  For example, if the truthset is
 a heterozygous call with both alleles non-reference (HET_VAR1_VAR2), and the callset is a heterozygous call with both alleles non-reference with one of the alternate alleles
 matching an alternate allele in the callset, we would have a true positive, false positive, and false negative.  The true positive is from the matching alternate alleles, the
 false positive is the alternate allele found in the callset but not found in the truthset, and the false negative is the alternate in the truthset not found in the callset.

 We also include a true negative in cases where the reference allele is found in both the truthset and callset.</td>
					</tr>
					</tbody>
				</table>

        <hr>
        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
        <hr>
        <p class="see-also">See also 
        	<a href="https://software.broadinstitute.org/gatk/documentation/">General Documentation</a> |
        	<a class="hide_me_php" href="index.html">Tool Docs Index</a> <a class="hide_me_html" href="index">Tool Documentation Index</a> |
        	<a href="https://gatkforums.broadinstitute.org/gatk/">Support Forum</a>
        </p>

        <p class="version">GATK version 4.5.0.0 built at Wed, 13 Dec 2023 17:42:57 -0500.
        </p>

	</div>

	<?php printFooter($module); ?>