<?php

    include '../../../common/include/common.php';
    include_once '../../config.php';
    $module = modules::GATK;
    printHeader($module, $name, topSN::guide);
?>

<link type='text/css' rel='stylesheet' href='gatkDoc.css'>

<div class='row-fluid' id="top">

	
	<?php $group = 'Metrics'; ?>

	<section class="span4">
		<aside class="well">
			<a href="index.html"><h4><i class='fa fa-chevron-left'></i> Back to Tool Docs Index</h4></a>
		</aside>
		<?php getForumPosts( 'GcBiasSummaryMetrics' ) ?>

	</section>

	<div class="span8">

		<h1>GcBiasSummaryMetrics</h1>

		<p class="lead">Metrics</p>

			<h3>Category
				<small> Metrics</small>
			</h3>
		<hr>
		<h2>Overview</h2>
		High level metrics that capture how biased the coverage in a certain lane is.
@author Tim Fennell

				<p>This table summarizes the values that are specific to this metric.</p>
				<table class="table table-striped table-bordered table-condensed">
					<thead>
					<tr>
						<th>Metric</th>
						<th>Summary</th>
					</tr>
					</thead>
					<tr>
						<td>ACCUMULATION_LEVEL<br />
						<td></td>
					</tr>
					<tr>
						<td>READS_USED<br />
						<td>This option is used to mark including or excluding duplicates.</td>
					</tr>
					<tr>
						<td>WINDOW_SIZE<br />
						<td>The window size on the genome used to calculate the GC of the sequence.</td>
					</tr>
					<tr>
						<td>TOTAL_CLUSTERS<br />
						<td>The total number of clusters that were seen in the gc bias calculation.</td>
					</tr>
					<tr>
						<td>ALIGNED_READS<br />
						<td>The total number of aligned reads used to compute the gc bias metrics.</td>
					</tr>
					<tr>
						<td>AT_DROPOUT<br />
						<td>Illumina-style AT dropout metric.  Calculated by taking each GC bin independently and calculating
 (%ref_at_gc - %reads_at_gc) and summing all positive values for GC=[0..50].</td>
					</tr>
					<tr>
						<td>GC_DROPOUT<br />
						<td>Illumina-style GC dropout metric.  Calculated by taking each GC bin independently and calculating
 (%ref_at_gc - %reads_at_gc) and summing all positive values for GC=[50..100].</td>
					</tr>
					<tr>
						<td>GC_NC_0_19<br />
						<td>Normalized coverage over quintile of GC content ranging from 0 - 19.</td>
					</tr>
					<tr>
						<td>GC_NC_20_39<br />
						<td>Normalized coverage over each quintile of GC content ranging from 20 - 39.</td>
					</tr>
					<tr>
						<td>GC_NC_40_59<br />
						<td>Normalized coverage over each quintile of GC content ranging from 40 - 59.</td>
					</tr>
					<tr>
						<td>GC_NC_60_79<br />
						<td>Normalized coverage over each quintile of GC content ranging from 60 - 79.</td>
					</tr>
					<tr>
						<td>GC_NC_80_100<br />
						<td>Normalized coverage over each quintile of GC content ranging from 80 - 100.</td>
					</tr>
					</tbody>
				</table>

        <hr>
        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
        <hr>
        <p class="see-also">See also 
        	<a href="https://software.broadinstitute.org/gatk/documentation/">General Documentation</a> |
        	<a class="hide_me_php" href="index.html">Tool Docs Index</a> <a class="hide_me_html" href="index">Tool Documentation Index</a> |
        	<a href="https://gatkforums.broadinstitute.org/gatk/">Support Forum</a>
        </p>

        <p class="version">GATK version 4.5.0.0 built at Wed, 13 Dec 2023 17:42:57 -0500.
        </p>

	</div>

	<?php printFooter($module); ?>